src/test/functional/tests/buzzPage.spec.ts

```diff
@@ -16,12 +18,23 @@ test.describe("Buzz Page", () => {
     await loginPage.loginUser(adminUserTestData.userName, adminUserTestData.password);
   });
 
-  test("Test Case 1: Share file and Delete file", async ({ page }) => {
+  test.afterEach(async ({ page }) => {
+    buzzPage = new BuzzPage(page);
+    await buzzPage.deletePhotos()
```

Comment:
deletePhotos should have a name that better describes what it does - basically something like deletePhotoInNewestPost (photo - singular). And please see the comments below about running in parallel - or even on an environment that you don't have complete control of - like an online version of the application.